ddk-configuration/pmd/ruleset-test.xml

```diff
@@ -1,240 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I would not remove this file even if the settings were the same like the normal ones. We often had cases where we wanted to have differences between normal and test code.

Reply:
In this case, I actually think that the 2 rules we ignore are useful even in tests - currently only "Shouldn't throw Exception" (true positives fixed) and "Assertions should have a message"(again, true positives fixed, and I think the tests are nicer for it)